src/Tools/MicroBuild/post-build.ps1

```diff
@@ -0,0 +1,79 @@
+# Run our post build steps.  This occurs after all of our shipping binaries are built and fully 
+# signed.  The steps we need to execute are:
+#
+#   1. Run GitLink on the PDBs so we have proper source linking.
+#   2. Setup the <Configuration>\Index directory for simple / efficient indexing in official builds
+
+
+param([string]$config = "Release")
+Set-StrictMode -version 2.0
+$ErrorActionPreference="Stop"
+
+# Get the path to all of the shipping EXE / DLL files.
+function Get-BinaryList() {
+    $list = @()
+    $config = Join-Path $repoDir "build\config\SignToolData.json"
+    $j = ConvertFrom-Json (Get-Content -raw $config)
+    foreach ($entry in $j.sign) {
+        foreach ($v in $entry.values) { 
+            $ext = [IO.Path]::GetExtension($v)
+            if (($ext -eq ".dll") -or ($ext -eq ".exe")) {
+                $path = Join-Path $configDir $v
+                $list += $path;
+            }
+        }
+    }
+
+    return $list
+}
+
+# Run GitLink on all of our PDB files in their original location.  
+function Run-GitLink($binaryLst) {
+
+    $gitlinkVersion = Get-PackageVersion "GitLink"
+    $gitlink = Join-Path (Get-PackagesDir) "GitLink\$gitlinkVersion\build\GitLink.exe"
+
+    Write-Host "Running GitLink"
+    foreach ($filePath in $binaryList) { 
+        $pdbPath = [IO.Path]::ChangeExtension($filePath, ".pdb")
+        Write-Host "`t$pdbPath"
+        Exec-Block { & $gitlink -u "https://github.com/dotnet/roslyn" $pdbPath --baseDir $repoDir } | Write-Host
+    }
+}
+
+# Create single directory to hold all of the items that we need to be indexed by 
```

Comment:
This concerns the MicroBuild Index Symbols step in our official build.  Today it is indexing virtually every binary that appears in our `Binaries\Release` folder.  Irrespective of whether or not we actually build or shipped it.  

Given the parameters to this build task it's not realistic to fix it with our present directory layout.  Much easier to create a simpler directory layout and then change that official build step to point at this new directory.